.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Probably we can use this https://github.com/github/gitignore/blob/main/Node.gitignore